src/Jwt/Symfony/Authentication/JsonWebToken.php

```diff
@@ -87,6 +90,34 @@ public function getUserId(): string
         return $this->token->getClaim('sub');
     }
 
+    public function containsUserIdentityDetails(): bool
+    {
+        // Tokens from the V1 JWT provider (= custom) and V2 JWT provider (= Auth0 ID tokens) contain the username and
+        // email. Auth0 access tokens do not.
+        return $this->hasClaims(['nick', 'email']) || $this->hasClaims(['nickname', 'email']);
```

Comment:
You could replace this with `return !is_null($this->getUserIdentityDetails());`.
That way we are sure the logic in both methods never drifts apart.

Reply:
Good point, but this method was not needed after all